installer/roles/image_build/templates/nginx.conf

```diff
@@ -36,7 +36,10 @@ http {
     }
 
     server {
-        listen 8052 default_server;
+        listen 8052 ssl default_server;
```

Comment:
Thanks for rebasing. As mentioned in the [referenced PR](https://github.com/ansible/awx/pull/2135#issuecomment-429138252), some folks are running AWX behind a load balancer and aren't doing https there. Since providing the certificate is optional, we probably only want to use https and the additional ssl settings if a certificate path is provided to the install playbook.

One way to do that would be to move this file into a jinja template that conditionally applies the additional `ssl` directives based on the value of the `ssl_certificate` variable.

Reply:
I added the condition as suggested. I also removed the default certificate generation, because ssl is off by default now.